build.sbt

```
@@ -350,8 +350,17 @@ val mtagsSettings = List(
     if (isCI) Nil
     // NOTE(olafur) pprint is indispensable for me while developing, I can't
     // use println anymore for debugging because pprint.log is 100 times better.
-    else
-      List("com.lihaoyi" %% "pprint" % "0.6.6")
+    else {
+      val scalaV = scalaVersion.value
+      val isScala3RC = scalaV.stripPrefix("3.0.0-").startsWith("RC")
+      if (isScala3RC) {
```

Let's maybe do then:
```suggestion
      if (scala3RCVersions.contains(scalaVersion.value)) Nil
      else List("com.lihaoyi" %% "pprint" % "0.6.6")
```

pprint will not work too great with `for3Use2_13` from what I remember. And we can put `scala3RCVersions` into a separate collection.